docs/sources/mimir/release-notes/v2.9.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Grafana Mimir version 2.9 release notes"
+menuTitle: "V2.9 release notes"
+description: "Release notes for Grafana Mimir version 2.9"
+weight: 300
+---
+
+# Grafana Mimir version 2.9 release notes
+
+Grafana Labs is excited to announce version 2.9 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Experimental support for optimized series selection strategy** Reduce the volume of data necessary to complete requests with individual label selectors that are present on a lot of series.
```

Comment:
I think this is not much clear. I would clarify:
1. This applies to metrics fetched from the long-term storage via the store-gateway
2. Expand what means "individual label selectors that are present on a lot of series"

Reply:
How about this? I think an example is easier to grasp than explaining exactly what this means.

```suggestion
- **Reduced store-gateway memory utilization on fetching series from long-term storage** Reduce the volume of data necessary to evaluate queries which select series by labels which are present on many series, such as `datacenter="nyc"`.
```

Reply:
How about
```suggestion
- **Experimental support for optimized series selection strategy** For queries with broad label matchers (e.g. `datacenter="dc1"), Mimir 2.9 will fetch a reduced amount of index data, which leads to a significant reduction in memory allocations in the store-gateway.
```